x-pack/plugins/enterprise_search/public/applications/app_search/components/schema/views/schema.test.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { EuiPageHeader, EuiButton } from '@elastic/eui';
+
+import { Schema } from './';
+
+describe('Schema', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders', () => {
+    const wrapper = shallow(<Schema />);
+
+    expect(wrapper.isEmptyRender()).toBe(false);
+    // TODO: Check for schema components
+  });
+
+  it('renders page action buttons', () => {
+    const wrapper = shallow(<Schema />)
```

Comment:
These buttons get passed in through the `rightSideItems` prop, right? I think I usually inspect that prop directly on the `EuiPageHeader` component to get reference to those buttons. I guess this is a little nicer because you're just rendering the header and then finding the buttons in the result.

Reply:
Haha yeah I think both are probably about the same / will get you the same results. It might be nice to DRY out a test helper for this some day 🤔